TaskManager.java

```diff
@@ -0,0 +1,70 @@
+@Component
+public class TaskManager {
```

Comment:
Is it high load project? Thousands of users?
If yes:
- Unit tests? (and maybe performance tests as well)
- Logs and metrics to control performance?